cmd/podman/create.go

```diff
@@ -932,3 +953,51 @@ func createContainerFromCreateConfig(r *libpod.Runtime, createConfig *cc.CreateC
 	}
 	return ctr, nil
 }
+
+func makeHealthCheckFromCli(c *cliconfig.PodmanCommand) (*manifest.Schema2HealthConfig, error) {
+	inCommand := c.String("healthcheck-command")
+	inInterval := c.String("healthcheck-interval")
+	inRetries := c.Uint("healthcheck-retries")
+	inTimeout := c.String("healthcheck-timeout")
+	inStartPeriod := c.String("healthcheck-start-period")
+
+	// Every healthcheck requires a command
+	if !c.Flags().Changed("healthcheck-command") {
+		return nil, errors.New("Must define a healthcheck command for all healthchecks")
+	}
+
+	hc := manifest.Schema2HealthConfig{
+		Test: strings.Split(inCommand, " "),
+	}
+	if c.Flags().Changed("healthcheck-interval") {
+		// TODO we should ack a minimum value and check against it
```

Comment:
I don't know if I agree on the minimum, since it would be difficult to say how much the minimum should be. If we don't have this `minimum` in buildah, then users could get around it.  Eliminate this comment.

Reply:
there is a minimum interval value set iirc. it is something in the ms range if i recall.